canopen/nmt.py

```diff
@@ -135,3 +137,98 @@ def wait_for_bootup(self, timeout=10):
 
 class NmtError(Exception):
     """Some NMT operation failed."""
+
+class NmtSlave(object):
+    """
+    Handles the NMT state and handles heartbeat NMT service.
+    """
+    def __init__(self, node_id, local_node):
+        self._id = node_id
+        self.network = None
+        self._state = 0
+        self._timer_thread = None
+        self._thread_stop = None
+        self._heartbeat_time_ms = 0
+        self._local_node = local_node
+
+    def on_command(self, can_id, data, timestamp):
+        (cmd, _) = struct.unpack_from("<BB", data)
+
+        logger.info("Node %d received command %d",can_id, cmd)
```

Comment:
I think `can_id` should be replaced with `self._id`.

Reply:
Fixed